editor/modes/visual-editor/block.js

```diff
@@ -159,14 +164,15 @@ class VisualEditorBlock extends wp.element.Component {
 				{ isSelected && ! isTyping &&
 					<div className="editor-visual-editor__block-controls">
 						<BlockSwitcher uid={ block.uid } />
-						{ !! settings.controls && (
+						{ toolbars.map( ( controls, index ) => (
```

Comment:
This will error if a block doesn't define controls because `toolbars` will be undefined. We'll want to restore the truthy test or use Lodash's `map`.